runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/util/BoundedQueueExecutor.java

```diff
@@ -73,7 +74,8 @@ public BoundedQueueExecutor(
           protected void beforeExecute(Thread t, Runnable r) {
             super.beforeExecute(t, r);
             synchronized (BoundedQueueExecutor.this) {
-              if (++activeCount >= maximumPoolSize && startTimeMaxActiveThreadsUsed == 0) {
+              if (++activeCount >= BoundedQueueExecutor.this.maximumPoolSize
```

Comment:
how about naming the method param initialMaximumPoolSize instead of using this.BQE.maximumPoolSize here?
Then there won't be the confusion and using initialMaximumPoolSize here would be obviously incorrect

Reply:
done